watertap/examples/flowsheets/lsrro/lsrro.py

```diff
@@ -1209,3 +1226,25 @@ def display_system(m):
 def display_RO_reports(m):
     for stage in m.fs.ROUnits.values():
         stage.report()
+
+
+if __name__ == "__main__":
+    m, results = run_lsrro_case(
+        number_of_stages=3,
+        water_recovery=0.50,
+        Cin=70,  # inlet NaCl conc kg/m3,
+        Qin=1e-3,  # inlet feed flowrate m3/s
+        Cbrine=None,  # brine conc kg/m3
+        A_case=ACase.optimize,
+        B_case=BCase.optimize,
+        AB_tradeoff=ABTradeoff.equality_constraint,
+        # A_value=4.2e-12, #membrane water permeability coeff m/s-Pa
```

Comment:
Should this be uncommented?

Reply:
I wanted to leave it for now so that it's a viewable choice. Whenever we improve documenting, could come back and fix.